peps/pep-0710.rst

```diff
@@ -118,9 +118,9 @@ may be present in a given ``.dist-info`` directory; installers MUST NOT add both
 The ``provenance_url.json`` JSON file MUST be a dictionary, compliant with
 :rfc:`8259` and UTF-8 encoded.
 
-If present, it MUST contain exactly two keys. The first one is ``url``, with
+If present, it MUST contain exactly three keys. The first one is ``url``, with
 type ``string``.  The second key MUST be ``archive_info`` with a value defined
-below.
+below. The third key MUST be ``index_url`` with type ``string``.
```

Comment:
Elsewhere in this PEP we reference using the PEP 610 Direct URL structure, and we're adding `index_url` here (which I think is correct). Do we need to make a reference in this doc that we'll need to update the Direct URL structure to include an optional element (`index_url`) for archives that only applies to PEP 710 usages?

Reply:
Good idea - let's see the [results of discussions on discuss.python.org](https://discuss.python.org/t/pep-710-recording-the-provenance-of-installed-packages/25428/24)